src/tfr_utilities/include/tfr_utilities/control_code.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Shared codes and for querying status from the control system
+ * */
+#ifndef CONTROL_CODE_H
+#define CONTROL_CODE_H
+
+#include <cstdint>
+
+namespace tfr_utilities
+{
+    namespace JointAngle
+    {
+        static const float ANGLE_TOLERANCE = 0.01;
+        //NOTES must match up with constants in model
+        static const float ARM_TURNTABLE_MAX = 6.28319;
+        static const float ARM_TURNTABLE_MIN = 0.0;
+        static const float ARM_LOWER_MAX = 1.69297;
+        static const float ARM_LOWER_MIN = 0.20944;
+        static const float ARM_UPPER_MAX = 1.8675;
+        static const float ARM_UPPER_MIN = 0.0;
+        static const float ARM_SCOOP_MAX = 1.3446;
+        static const float ARM_SCOOP_MIN = -0.872665;
+        static const float BIN_MAX = 0.785398;
+        static const float BIN_MIN = 0.0;
+    }
```

Comment:
I don't suppose there's a way to load the constants into here and the model from one place, or load from the model into here?

Reply:
I just feel like it's a bug waiting to happen. We get the arm onto the robot and refine some of the constants to match real life but then they don't get changed in both places...

Reply:
It could be, but the overal geometry of the arm will not change frequently that would require a cutting torch and a very grumpy EE.

Reply:
I can make a maintenance card for it, if we have some time and or figure out a better way to go about it during the arm we can do that.